pkg/ddevapp/global_dotddev_assets/commands/host/dbeaver

```diff
@@ -5,7 +5,7 @@
 ## Usage: dbeaver
 ## Example: "ddev dbeaver [db] [user]"
 ## OSTypes: darwin,linux
-## HostBinaryExists: /Applications/DBeaver.app,/usr/bin/dbeaver,/usr/bin/dbeaver-ce,/usr/bin/dbeaver-le,/usr/bin/dbeaver-ue,/usr/bin/dbeaver-ee,/var/lib/flatpak/exports/bin/io.dbeaver.DBeaverCommunity,/snap/bin/dbeaver-ce
+## HostBinaryExists: /Applications/DBeaver.app,/usr/bin/dbeaver,/usr/bin/dbeaver-ce,/usr/bin/dbeaver-le,/usr/bin/dbeaver-ue,/usr/bin/dbeaver-ee,/var/lib/flatpak/exports/bin/io.dbeaver.DBeaverCommunity,/snap/bin/dbeaver-ce,$HOME/.local/share/flatpak/app/io.dbeaver.DBeaverCommunity/current/active/export/bin/io.dbeaver.DBeaverCommunity
```

Comment:
Thanks, were you able to test this manually? I wouldn't have necessarily expected that $HOME would be replaced in this context.